src/traveler.js

```diff
@@ -17,9 +17,10 @@ class Traveler {
   determineTrips(trips, currentDate, allDestinations) {
     trips.forEach(trip => {
       let tripDestination
-      allDestinations.forEach(destination => {
-        if (trip.destinationID === destination.id)
-        tripDestination = destination
+      allDestinations.find(destination => {
```

Comment:
what does the `find` prototype method return?

Appears you may be doing more than is necessary here...